### PR TITLE
[FIX] base: nl2br utility method in qweb should produce a Markup result

### PR DESCRIPTION
**Pull request by jco-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jco-odoo** [comment]
@Gorash Was this on purpose or an oversight?

**Comment by lordkrandel** [comment]
Bump.

File: odoo/addons/base/models/ir_qweb_fields.py [hunk 1]
```diff
@@ -26,7 +26,9 @@ def nl2br(string):
     :param str string:
     :rtype: unicode
     """
-    return pycompat.to_text(string).replace('\n', Markup('<br>\n'))
+    string = pycompat.to_text(string)
+    string_split = string.split('\n')
```
Comment (lordkrandel) [inline odoo/addons/base/models/ir_qweb_fields.py:30]:
```suggestion
    string_split = pycompat.to_text(string).split('\n')
```

**Comment by lordkrandel** [comment]
Bump for @jco-odoo to make the changes needed.

**Comment by lordkrandel** [comment]
Nvm, done by the framework team. Closing this.
https://github.com/odoo/odoo/pull/166535

**Pull request closed without merging**